roxie/ccd/ccdkey.cpp

```diff
@@ -76,12 +76,10 @@ class PtrToOffsetMapper
         // MORE - could cache last hit
         unsigned int a = 0;
         int b = numBases;
-        int rc;
         while ((int)a<b)
         {
             int i = (a+b+1)/2;
-            rc = pos - fragments[i-1].baseOffset;
-            if (rc>=0)
+            if (pos >= fragments[i-1].baseOffset)
```

Comment:
If fragments were > 2^31 in size then this could have caused the wrong fragment to be matched.